Lombiq.ChartJs.Samples/Recipes/Lombiq.ChartJs.Samples.Content.recipe.json

```diff
@@ -0,0 +1,2671 @@
+{
+  "name": "Lombiq.ChartJs.Samples.Content",
+  "displayName": "Lombiq Chart.js - Sample Content - Income/Expense",
+  "description": "Sample content for testing and demonstration of the Lombiq Chart.js module.",
+  "author": "Lombiq Technologies",
+  "website": "https://github.com/Lombiq/Orchard-Chart.js",
+  "version": "1.0",
+  "issetuprecipe": false,
+  "categories": [
+    "training"
+  ],
+  "tags": [
+    "developer",
+    "training",
+    "sample"
+  ],
+  "variables": {
+    "incomeTagsTaxonomyId": "taxonomyincometags00000000",
+    "expenseTagsTaxonomyId": "taxonomyexpensetags0000000"
+  },
+  "steps": [
+    {
+      "name": "feature",
+      "enable": [
+        "Lombiq.ChartJs.Samples"
+      ]
+    },
+    {
+      "name": "content",
+      "data": [
+        {
+          "ContentItemId": "[js: variables('expenseTagsTaxonomyId')]",
+          "ContentType": "Taxonomy",
+          "DisplayText": "Expense tags",
+          "Latest": true,
+          "Published": true,
+          "Owner": "[js: parameters('AdminUserId')]",
+          "Author": "[js: parameters('AdminUsername')]",
+          "TitlePart": {
+            "Title": "Expense tags"
+          },
+          "AliasPart": {
+            "Alias": "expense-tags"
+          },
+          "AutoroutePart": {
+            "Path": "expense-tags",
+            "SetHomepage": false,
+            "Disabled": false,
+            "RouteContainedItems": false,
+            "Absolute": false
+          },
+          "TaxonomyPart": {
+            "TermContentType": "Tag",
+            "Terms": [
+              {
+                "ContentItemId": "4pk4cy35x5ajyw03jnvhf4vpnm",
+                "ContentItemVersionId": null,
+                "ContentType": "Tag",
+                "DisplayText": "Household",
+                "Latest": false,
+                "Published": false,
+                "ModifiedUtc": "2022-05-01T06:35:38.8285335Z",
+                "PublishedUtc": null,
+                "CreatedUtc": null,
+                "Owner": null,
+                "Author": "[js: parameters('AdminUsername')]",
+                "TagPart": {},
```

Comment:
Now that you've removed the TagPart, please also remove these props from the recipe.